api/chaos/experimentation/v1/experiment_run_config_pair_details.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package clutch.chaos.experimentation.v1;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+
+import "chaos/experimentation/v1/form.proto";
+
+option go_package = "experimentationv1";
+
+message ExperimentRunConfigPairDetails {
```

Comment:
any reason this is in its own proto?

Reply:
just to avoid polluting the 'main' proto - do you think it should be a  part of the main experimentation file?

Reply:
I would prefer to keep it where it is for now since experimentation.proto is getting big anyway. Let me know if you have as strong opinion here.

Reply:
no strong opinion